css-brochure-form/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 60rem;
+  margin: auto;
+  line-height: 1.5rem;
+}
+
+h2 {
+  color: #ed645e;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: 500;
+  text-align: center;
+  letter-spacing: 0;
+}
+
+div input {
+  border-radius: 0.5em;
+  width: 42rem;
+  height: 3.5rem;
+  border: 1px solid #999;
+  margin-bottom: 1rem;
+  margin-left: 10rem;
+  margin-right: 10rem;
```

Comment:
Oh man, this is a **_lot_** of margin. What is the purpose of this? You could potentially just make the width of the `form` 42rem, then make this `width: 100%`